docs/docs-content/cluster-templates/cluster-templates.md

```diff
@@ -0,0 +1,138 @@
+---
+sidebar_label: "Cluster Templates"
+title: "Cluster Templates"
+description:
+  "Learn how Palette uses cluster templates to bring consistency to clusters and manage the cluster lifecycle."
+hide_table_of_contents: false
+sidebar_position: 0
+sidebar_custom_props:
+  icon: "object-ungroup"
+tags: ["cluster templates", "templates", "policies"]
+---
+
+:::preview
+
+:::
+
+Cluster templates are reusable blueprints that define and enforce the desired state and lifecycle of clusters deployed
+with Palette or [Palette VerteX](../vertex/vertex.md).
+
+Unlike [cluster profiles](../profiles/cluster-profiles/cluster-profiles.md), which define the cluster's software stack
+(including OS, Kubernetes distribution, network, storage, and add‑ons), cluster templates are a higher level abstraction
+that define the governance stack. They reference existing cluster profiles and operational
+[policies](./create-cluster-template-policies/create-cluster-template-policies.md), such as
+[maintenance policies](./create-cluster-template-policies/maintenance-policy.md), and leverage
+[cluster profile variables](../profiles/cluster-profiles/create-cluster-profiles/define-profile-variables/define-profile-variables.md)
+and [macros](../clusters/cluster-management/macros.md), allowing you to deploy, manage, and scale a synchronized fleet
+of clusters with minimal effort while configuring environment-specific values where needed.
+
+Cluster templates can be created at both the tenant and project scope. Cluster templates do not embed cluster profile
+and policy configurations but reference them as objects, allowing you to edit and replace them as needed.
+
+Cluster templates can be managed using [Terraform](../automation/terraform/terraform.md),
+[Crossplane](../automation/crossplane/crossplane.md), the [Palette Go SDK](../automation/palette-sdk/palette-sdk.md),
+and [APIs](/api/introduction).
+
+![Diagram of cluster template architecture](/cluster-templates_diagram.webp)
+
+## Cluster Profile Behavior
+
+Cluster templates link to [cluster profiles](../profiles/cluster-profiles/cluster-profiles.md), which remain the
+foundation of deploying clusters in Palette. Each cluster template must link to one
+[full](../profiles/cluster-profiles/create-cluster-profiles/create-full-profile.md) or
+[infrastructure profile](../profiles/cluster-profiles/create-cluster-profiles/create-infrastructure-profile.md) and one
+[maintenance policy](./create-cluster-template-policies/maintenance-policy.md). You can reference as many
+[add-on profiles](../profiles/cluster-profiles/create-cluster-profiles/create-addon-profile/create-addon-profile.md) as
+desired. Only one policy of each type can be linked per cluster template (for example, one maintenance policy per
+template).
+
+Once a cluster profile is linked to a cluster template, that version of the cluster profile becomes immutable.
```

Comment:
```suggestion
Once a cluster profile version is linked to a cluster template, that version of the cluster profile becomes immutable.
```

This should be emphasized a few lines above, that you link a cluster profile VERSION and not the entire cluster profile.

Reply:
I've clarified this above now: https://github.com/spectrocloud/librarium/pull/8323/files#r2440111708